src/interfaces/IWhitelist.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+interface IWhitelist {
+    function sanctioned(address _subAccount) external view returns (bool);
+
+    function grappaAccess(address _subAccount) external view returns (bool);
```

Comment:
better name to `engineAccess` or just `access`, `permissioned`
Otherwise kind of imply this is the access control on Grappa contract

Reply:
yeah, good call